src/codegen/fast_math.hpp

```diff
@@ -214,4 +214,14 @@ static inline float exprelr(float x) {
     }
 
     return x / vexpm1(x);
-}
\ No newline at end of file
+}
+
+/// double precision log10 function
+inline double log10(double f) noexcept {
+    return log(f) / log(10.0);
```

Comment:
for efficiency I would write `1/log(10.0)` as a `constexpr` and remove the runtime division

Reply:
I thought that later actually and I was planning to do that. Thanks for suggestion! :+1:

Reply:
+1